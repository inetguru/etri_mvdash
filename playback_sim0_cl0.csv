Seg	vp	Start	Bffer?	qV0	qV1	qV2	qV3	qV4
0	0	452185	0	51	51	51	51	51
1	0	2452185	0	42	0	0	0	0
2	0	4452185	0	96	0	0	0	0
3	0	6452185	0	96	0	0	0	0
4	0	8452185	0	93	0	0	0	0
5	0	10452185	0	89	0	0	0	0
6	0	12452185	0	92	0	0	0	0
7	0	14452185	0	92	0	0	0	0
8	0	16452185	0	93	6	6	6	6
9	0	18452185	0	93	7	7	7	7
10	0	20452185	0	95	7	7	7	7
11	0	22452185	0	91	4	4	4	4
12	0	24452185	0	90	0	0	0	0
13	0	26452185	0	92	1	1	1	1
14	0	28452185	0	92	2	2	2	2
15	0	30452185	0	90	0	0	0	0
16	0	32452185	0	92	1	1	1	1
17	0	34452185	0	94	0	0	0	0
18	0	36452185	0	94	0	0	0	0
19	0	38452185	0	94	0	0	0	0
20	0	40452185	0	93	0	0	0	0
21	0	42452185	0	96	0	0	0	0
22	0	44452185	0	97	2	2	2	2
23	0	46452185	0	96	1	1	1	1
