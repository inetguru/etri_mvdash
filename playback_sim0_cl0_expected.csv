Seg,vp,Start,Bffer?,qV0,qV1,qV2,qV3,qV4
0,0,410188,0,0,0,0,0,0
1,0,2410188,0,0,0,0,0,0
2,0,4410188,0,5,0,0,0,0
3,0,6410188,0,5,0,0,0,0
4,2,8410188,0,5,0,0,0,0
5,2,10410188,0,0,0,5,0,0
6,2,12410188,0,0,0,5,0,0
7,2,14410188,0,0,0,5,0,0
8,2,16410188,0,0,0,5,0,0
9,4,18410188,0,0,0,5,0,0
10,4,20410188,0,0,0,0,0,5
11,4,22410188,0,0,0,0,0,5
12,4,24410188,0,0,0,0,0,5
15,0,26410188,0,0,0,0,0,5
14,0,28410188,0,5,0,0,0,0
15,0,30410188,0,5,0,0,0,0
16,0,32410188,0,5,0,0,0,0
17,1,34410188,0,5,0,0,0,0
18,1,36410188,0,0,5,0,0,0
19,0,38410188,0,0,5,0,0,0
20,0,40410188,0,5,0,0,0,0
21,0,42410188,0,5,0,0,0,0
22,0,44410188,0,5,0,0,0,0
23,0,46410188,0,5,0,0,0,0
